recipes/gtest/all/conanfile.py

```diff
@@ -31,6 +31,8 @@ def _minimum_compilers_version(self):
             return {
                 "Visual Studio": "14"
             }
```

Comment:
Could you please check that this is needed?

Reply:
https://github.com/google/googletest/tree/release-1.8.1#windows-requirements

Reply:
👍

Reply:
I would say they didn't change any requirement. If we gonna change it, please, do it for all versions, you can remove that conditional and use "4.8"

Reply:
I did a quick test on Linux + GCC 4.8 and it works.

Reply:
I'm not sure what you mean? 1.10.0 and newer work on 4.8?